Unable to install this package via npm

### What happened?

I am getting `Error: Cannot find module '/Users/ikovac/Projects/personal/pulumi-lambda/node_modules/@pulumi/archive/scripts/install-pulumi-plugin.js` error while trying to install this package.

### Expected Behavior

Installation should pass

### Steps to reproduce

npm i @pulumi/archive

### Output of `pulumi about`

CLI
Version      3.74.0
Go Version   go1.20.5
Go Compiler  gc

Plugins
NAME    VERSION
aws     5.42.0
awsx    1.0.2
docker  3.6.1
nodejs  unknown

Host
OS       darwin
Version  12.6
Arch     arm64

This project is written in nodejs: executable='/Users/ikovac/.nvm/versions/node/v18.16.0/bin/node' version='v18.16.0'

Current Stack: ikovac/pulumi-lambda/dev

Found no resources associated with dev

Found no pending operations associated with dev

Backend
Name           pulumi.com
URL            https://app.pulumi.com/ikovac
User           ikovac
Organizations  ikovac

Dependencies:
NAME            VERSION
@pulumi/aws     5.42.0
@pulumi/awsx    1.0.2
@pulumi/pulumi  3.76.0
@types/node     16.18.39

Pulumi locates its logs in /var/folders/mm/j4bk5d553pj6h4n8q4gk270r0000gn/T/ by default


### Additional context

_No response_

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -359,7 +359,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Publish SDKs
-      uses: pulumi/pulumi-package-publisher@v0.0.7
+      uses: pulumi/pulumi-package-publisher@v0.0.9
     - env:
         SLACK_CHANNEL: provider-upgrade-publish-status
         SLACK_COLOR: "#FF0000"

--- a/.github/workflows/master.yml
+++ b/.github/workflows/master.yml
@@ -359,7 +359,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Publish SDKs
-      uses: pulumi/pulumi-package-publisher@v0.0.7
+      uses: pulumi/pulumi-package-publisher@v0.0.9
     - env:
         SLACK_CHANNEL: provider-upgrade-publish-status
         SLACK_COLOR: "#FF0000"

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -321,7 +321,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Publish SDKs
-      uses: pulumi/pulumi-package-publisher@v0.0.7
+      uses: pulumi/pulumi-package-publisher@v0.0.9
     - env:
         SLACK_CHANNEL: provider-upgrade-publish-status
         SLACK_COLOR: "#FF0000"

